### PR TITLE
Add support for SAUCE_USERNAME and SAUCE_ACCESS_KEY

### PR DESCRIPTION
**Pull request by FriggaHel**
# One-line summary

Add support for `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY`

## Description

Add support for `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY`

## Types of Changes

- New feature

### CODE REVIEW

File: visual-dotnet/SauceLabs.Visual/VisualClient.cs [hunk 1]
```diff
@@ -46,6 +65,11 @@ public VisualClient(WebDriver wd, Region region, string username, string accessK
         /// <param name="buildOptions">the options of the build creation</param>
         public VisualClient(WebDriver wd, Region region, string username, string accessKey, CreateBuildOptions buildOptions)
         {
+            if (string.IsNullOrWhiteSpace(username) || string.IsNullOrEmpty(accessKey))
```
Comment (omacranger) [inline visual-dotnet/SauceLabs.Visual/VisualClient.cs:68]: Should these two `IsNull...` checks be the same?

Comment (FriggaHel) [reply visual-dotnet/SauceLabs.Visual/VisualClient.cs:68]: They definitively should !

**Review by ebekebe — APPROVED** [review]

File: visual-dotnet/SauceLabs.Visual/VisualClient.cs [hunk 2]
```diff
@@ -47,6 +66,11 @@ public VisualClient(WebDriver wd, Region region, string username, string accessK
         /// <param name="buildOptions">the options of the build creation</param>
         public VisualClient(WebDriver wd, Region region, string username, string accessKey, CreateBuildOptions buildOptions)
```
Comment (konraddysput) [inline visual-dotnet/SauceLabs.Visual/VisualClient.cs:67]: nice to have: 
we need to add a description for rest parameters here.

Comment (FriggaHel) [reply visual-dotnet/SauceLabs.Visual/VisualClient.cs:67]: Which rest parameters are you mentioning ?

**Pull request merged**